src/main/java/de/retest/recheck/ui/diff/AttributeDifference.java

```diff
@@ -65,6 +69,10 @@ public String getKey() {
 		return key;
 	}
 
+	public void setWarning( final Warning warning ) {
+		this.warning = warning;
```

Comment:
Is the setter necessary? I would prefer this thing to be `final`.

Reply:
With "this" I mean the field, not the setter. Sorry for confusing you. So, can the field `warning` be final and the setter be removed?

Reply:
The setter method will be used later in the recheck-web project. Or you prefer another method to be implemented for this?

Reply:
You can see the usage of setWarning method in the link below:
https://github.com/retest/recheck-web/pull/251